app/views/returns/NonExportedHumanMedicinesPlasticPackagingWeightView.scala.html

```diff
@@ -53,6 +53,7 @@ <h1 class="govuk-heading-l">@messages("nonExportedHumanMedicinesPlasticPackaging
             .asKg()
             .withWidth(Fixed10)
             .withHint(Hint(content = Text(messages("nonExportedHumanMedicinesPlasticPackagingWeight.hint"))))
+            .withAttribute("aria-label" -> "human-medicine-input")
```

Comment:
should this be aria-labelledby  ?

Reply:
@liampan can be either https://dequeuniversity.com/rules/axe/4.3/label?application=webdriverjs

Reply:
resolved. removed and add the for Id